src/spec.c

```diff
@@ -1181,10 +1155,6 @@ StrongRef IndexSpec_Parse(const char *name, const char **argv, int argc, QueryEr
     SchemaRule_FilterFields(spec);
   }
 
-  for (int i = 0; i < spec->numFields; i++) {
-    FieldsGlobalStats_UpdateStats(spec->fields + i, 1);
-  }
```

Comment:
Why removing this?

Reply:
Moved into the function that actually adds the fields